config/demo.exs

```diff
@@ -0,0 +1,27 @@
+import Config
+
+config :wanda, Wanda.Policy, execution_server_impl: Wanda.Executions.FakeServer
+
+config :wanda, WandaWeb.Endpoint,
+  check_origin: :conn,
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  server: true
+
+config :logger, level: :debug
+
+config :wanda, Wanda.Messaging.Adapters.AMQP,
```

Comment:
Do we need this messaging configuration? It is the same as in the `config.exs` file

Reply:
You are right, I think we can let it go. Since I was making basically a clone of the prod env but with the faker, I copied this from there too. It probably means we can also remove this from `prod.exs`